tests/Feature/AcceptSubmissionTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+use App\Enums\SubmissionStatuses;
+use App\Models\Submission;
+use App\Models\User;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+
+uses(RefreshDatabase::class);
+
+test('not able to accept submission if not logged user', function () {
```

Comment:
Just linking some resources that will make test look cleaner and faster to write. 
Testing conventions (mainly for test names): https://github.com/framgia/laravel-test-guideline/blob/master/en/Conventions.md

Data providers: https://dev.to/arielmejiadev/phpunit-dataprovider-with-laravel-9gg